### PR TITLE
Create-animals-table

### PR DESCRIPTION
**Pull request by Patfarmurs**
Screenshot-2

Screenshot-1
.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,192 +57,91 @@ After you're finished please remove all the comments and instructions!
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (Shedrack-Sunday) [inline README.md:141]: - [x] Kindly add a License file to the root of your project 👍 
- Including a license file in a README is important as it clarifies the terms and conditions under which the software can be used, providing legal protection and promoting open collaboration while preventing potential misuse or infringement.

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Patfarmurs   👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudus on the SQL queries ✔️ 
  
# Required Changes  🌵 

- [ ] You made some impressive implementations which I must say is a job well done. However, there are a few  I suggest   you make In line with the requirement


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (Patfarmurs) [reply README.md:141]: Hi @Shedrack-Sunday.
MIT LICENSE is there and the link is working correctly.

**Review by codecaiine — APPROVED** [review]
Hi @Patfarmurs , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Good use of Gitflow ✔️
SQL queries are well setup✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Patfarmurs** [comment]
Thank you @codecaiine.

**Pull request merged**